R/PipeOpFFS.R

```diff
@@ -140,17 +182,22 @@ make_fextractor = function(f) {
       upper = interval[[2L]]
 
       if (is.na(lower) || is.na(upper)) {
-        return(rep(NA_real_, length(x))) # no observation in the given interval [left, right]
+        res = map(features, function(f) {
+          rep(NA_real_, length(x)) # no observation in the given interval [left, right]
+        })
+        return(res)
       }
 
-      res = map_dbl(seq_along(x), function(i) {
+      res = map(seq_along(x), function(i) {
         value = tf::tf_evaluations(x[i])[[1L]]
```

Comment:
I did a small benchmark that showed that first getting all evaluations (`tf_evaluations(x)`) and then subsetting is faster than doing it that way, probably because of the additional overhead of creating the new tf vector first

Reply:
please also change this in the reg case